modules/ppcp-api-client/src/Endpoint/IdentityToken.php

```diff
@@ -84,7 +94,9 @@ public function generate_for_customer( int $customer_id ): Token {
 				'Content-Type'  => 'application/json',
 			),
 		);
-		if ( $customer_id && defined( 'PPCP_FLAG_SUBSCRIPTION' ) && PPCP_FLAG_SUBSCRIPTION ) {
+		if ( $customer_id
+			&& ( $this->settings->has( 'vault_enabled' ) && $this->settings->get( 'vault_enabled' ) )
+			&& defined( 'PPCP_FLAG_SUBSCRIPTION' ) && PPCP_FLAG_SUBSCRIPTION ) {
 			$args['body'] = wp_json_encode( array( 'customer_id' => $this->prefix . $customer_id ) );
```

Comment:
`$args...` looks like part of the condition.

I think phpcs rules allow something like
```
		if (
			array_key_exists( $field_id, $payer_phone_map )
			&& $payer
			&& $payer->phone()
		) {
```

which is more readable.